messaging-experiments.json

```diff
@@ -1146,5 +1146,488 @@
         }
       ]
     }
+  },
+  {
+    "id": "MESSAGE_ROUTER_PICTURE_IN_PICTURE_CFR",
+    "enabled": true,
+    "filter_expression": "env.channel == 'release' && env.version >= '78.' && env.version < '79.'",
+    "targeting": "locale == 'en-US' && userPrefs.cfrFeatures &&\n'media.videocontrols.picture-in-picture.enabled'|preferenceValue &&\n[userId, 'messaging-experiments-cfr']|bucketSample(5312, 650, 10000) &&\n((currentDate|date - profileAgeCreated) / 86400000) > 7\n",
```

Comment:
Doesn't exclude people that opted out of studies `app.shield.optoutstudies.enabled`